website/content/docs/post-processors/vagrant-cloud.mdx

```diff
@@ -1,211 +0,0 @@
----
```

Comment:
The removed content files have been moved to [`hashicorp/packer-plugin-vagrant`](https://github.com/hashicorp/packer-plugin-vagrant), so I think they can be deleted from the `packer` repo.

If left in the repo, the "unlinked content" check fails, because it thinks these files should appear somewhere in navigation so that they can be published (rather than sitting unlinked and unpublished in the content folder as they currently do).